src/loader-components/clinical-lowlight-theme/ExampleTemplate.module.scss

```diff
@@ -1,6 +1,10 @@
 :local{
   .clinical-lowlight-theme {
+    --terra-dev-site-example-template-hover-selected-code-toggle-background-color: #505a5f;
+    --terra-dev-site-example-template-selected-code-toggle-background-color: #3f474b;
+    --terra-dev-site-example-template-hover-code-toggle-background-color: #373d41;
     --terra-dev-site-example-template-code-toggle-background-color: #2e3539;
```

Comment:
nit: keep the theme variables in the same order for both files

Reply:
Reorganized: [f086e80](https://github.com/cerner/terra-dev-site/pull/281/commits/f086e8097bc5393e9187cd5ca9efb848e1dcb773).